index.html

```diff
@@ -1,29 +1,118 @@
+  </div>
+  
+  <div class="panel">
+    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed auctor, velit vel malesuada dictum, velit velit congue metus, in congue magna ipsum a neque. 
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed auctor, velit vel malesuada dictum, velit velit congue metus, in congue magna ipsum a neque. 
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed auctor, velit vel malesuada dictum, velit velit congue metus, in congue magna ipsum a neque. 
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed auctor, velit vel malesuada dictum, velit velit congue metus, in congue magna ipsum a neque.
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed auctor, velit vel malesuada dictum, velit velit congue metus, in congue magna ipsum a neque.
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed auctor, velit vel malesuada dictum, velit velit congue metus, in congue magna ipsum a neque.
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed auctor, velit vel malesuada dictum, velit velit congue metus, in congue magna ipsum a neque.</p>
+  </div>
+  <div class="subpanel-container">
+    <div class="subpanel">
+      <p>Subpanel 1: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed auctor, velit vel malesuada dictum, velit velit congue metus, in congue magna ipsum a neque.</p>
+    </div>
+    <div class="subpanel">
+      <p>Subpanel 2: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed auctor, velit vel malesuada dictum, velit velit congue metus, in congue magna ipsum a neque.</p>
+    </div>
+    <div class="subpanel">
+      <p>Subpanel 3: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed auctor, velit vel malesuada dictum, velit velit congue metus, in congue magna ipsum a neque.</p>
+    </div>
+    <div class="subpanel">
+      <p>Subpanel 4: Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed auctor, velit vel malesuada dictum, velit velit congue metus, in congue magna ipsum a neque.</p>
+    </div>
+  </div>
+  <div class="panel">
+    <table>
+        <tr>
+            <th style="padding: 10px">Developer</th>
+            <th style="padding: 10px">Developer</th>
+            <th style="padding: 10px">Developer</th>
+            <th style="padding: 10px">Developer</th>
+            <th style="padding: 10px">Developer</th>
+            <th style="padding: 10px">Developer</th>
+        </tr>
+        <tr>
+            <td>Data 1</td>
+            <td>Data 3</td>
+            <td>Data 5</td>
+            <td>Data 7</td>
+            <td>Data 9</td>
+            <td>Data 11</td>
+        </tr>
+        <tr>
+            <td>Data 2</td>
+            <td>Data 4</td>
+            <td>Data 6</td>
+            <td>Data 8</td>
+            <td>Data 10</td>
+            <td>Data 12</td>
+        </tr>
+    </table>
```

Comment:
This would be much better done using CSS grid or flex styles as opposed to `table` layout. 

Using `table` s for layouts introduces a host of issues. 

Ideally `table` elements should be used just for tabular data. The way this content is laid out would be much more accessible and suitable in `div` elements with `hX` and flex or grid styles.